packages/frontend-api/src/Model/Token/TokenAuthenticator.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Token;
+
+use InvalidArgumentException;
+use Shopsys\FrameworkBundle\Model\Customer\User\CustomerUser;
+use Shopsys\FrameworkBundle\Model\Customer\User\FrontendCustomerUserProvider;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\Component\Security\Core\User\UserProviderInterface;
+use Symfony\Component\Security\Guard\AbstractGuardAuthenticator;
+
+class TokenAuthenticator extends AbstractGuardAuthenticator
+{
+    protected const HEADER_AUTHORIZATION = 'Authorization';
+
+    /**
+     * @var \Shopsys\FrontendApiBundle\Model\Token\TokenFacade
+     */
+    protected $tokenFacade;
+
+    /**
+     * @param \Shopsys\FrontendApiBundle\Model\Token\TokenFacade $tokenFacade
+     */
+    public function __construct(TokenFacade $tokenFacade)
+    {
+        $this->tokenFacade = $tokenFacade;
+    }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @return bool
+     */
+    public function supports(Request $request): bool
+    {
+        return $request->headers->has(static::HEADER_AUTHORIZATION);
+    }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @return string|null
+     */
+    public function getCredentials(Request $request): ?string
+    {
+        return $request->headers->get(static::HEADER_AUTHORIZATION);
```

Comment:
I noticed you changed the header. What I read, the authorization header can be used for basic and token auth, so it's usually in the following form `Authorization: Bearer <api-key>`.

Is it right to let it without the Bearer claim?
https://learning.postman.com/docs/postman/sending-api-requests/authorization/#bearer-token

Reply:
You are right, sorry - 30d7cdd6bb1753470e5fb2940fe7ca655e4a30d9